bindings/python/docs/source/conf.py

```diff
@@ -23,7 +23,7 @@
 html_show_sphinx = False
 
 # The full version, including alpha/beta/rc tags
-version = '0.2.0.dev0'
```

Comment:
Why do we need to duplicate the version here?

Reply:
To display in the docs. In PyMongo we don't need to do this because the package can be installed and the version imported... I don't think that will work here.

Reply:
Can you add a comment here and in `version.py` to make sure these two stay in sync?

Reply:
Actually, I remembered a way I handled this in another project... See the next commit.

Reply:
Nice!